test/unit/test_idstorage.c

```diff
@@ -0,0 +1,71 @@
+#include <r_util.h>
+#include "minunit.h"
+
+bool test_r_id_storage_add(void) {
+	char *str = "lol";
+	RIDStorage *ids = r_id_storage_new (5, 23);
+	ut32 id;
+	r_id_storage_add (ids, str, &id);
```

Comment:
Check return value

Reply:
It is valid to add NULL?

Reply:
Do we have an R_IO_INVALID_ID UT32_MAX?

Reply:
Adding NULL to the storage can be done in theory, but you should do that if you want to use iteration functions

Reply:
and yes that limitiation of max io id exists, but it's not part of this test